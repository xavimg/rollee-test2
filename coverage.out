mode: set
words/internal/service/word.go:13.58,15.2 1 1
words/internal/service/word.go:17.51,20.2 2 1
words/internal/service/word.go:22.79,25.2 2 1
words/internal/storage/in_memory.go:33.64,42.2 3 1
words/internal/storage/in_memory.go:44.51,49.35 3 1
words/internal/storage/in_memory.go:55.2,57.12 3 1
words/internal/storage/in_memory.go:49.35,50.16 1 1
words/internal/storage/in_memory.go:50.16,52.4 1 1
words/internal/storage/in_memory.go:60.77,66.40 5 1
words/internal/storage/in_memory.go:72.2,72.19 1 1
words/internal/storage/in_memory.go:76.2,77.21 2 1
words/internal/storage/in_memory.go:66.40,67.58 1 1
words/internal/storage/in_memory.go:67.58,70.4 2 1
words/internal/storage/in_memory.go:72.19,74.3 1 1
words/internal/storage/in_memory.go:83.49,86.6 3 1
words/internal/storage/in_memory.go:86.6,87.10 1 1
words/internal/storage/in_memory.go:88.19,92.35 4 1
words/internal/storage/in_memory.go:97.4,97.18 1 1
words/internal/storage/in_memory.go:92.35,93.32 1 1
words/internal/storage/in_memory.go:93.32,95.6 1 1
words/internal/handlers/word.go:32.54,34.2 1 1
words/internal/handlers/word.go:36.71,39.60 2 1
words/internal/handlers/word.go:47.2,47.52 1 1
words/internal/handlers/word.go:53.2,53.35 1 1
words/internal/handlers/word.go:39.60,41.55 2 0
words/internal/handlers/word.go:41.55,44.4 2 0
words/internal/handlers/word.go:47.52,51.3 3 1
words/internal/handlers/word.go:56.84,59.62 2 1
words/internal/handlers/word.go:66.2,67.16 2 1
words/internal/handlers/word.go:73.2,73.23 1 1
words/internal/handlers/word.go:59.62,61.57 2 0
words/internal/handlers/word.go:61.57,63.4 1 0
words/internal/handlers/word.go:67.16,71.3 3 1
words/internal/handlers/word.go:77.66,79.16 2 1
words/internal/handlers/word.go:82.2,82.14 1 1
words/internal/handlers/word.go:85.2,85.18 1 1
words/internal/handlers/word.go:79.16,81.3 1 0
words/internal/handlers/word.go:82.14,84.3 1 1
