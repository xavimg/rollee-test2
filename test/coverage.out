mode: set
words/internal/http/handlers/word.go:31.51,33.2 1 1
words/internal/http/handlers/word.go:35.71,38.60 2 1
words/internal/http/handlers/word.go:38.60,42.3 3 0
words/internal/http/handlers/word.go:44.2,44.52 1 1
words/internal/http/handlers/word.go:44.52,48.3 3 0
words/internal/http/handlers/word.go:50.2,50.35 1 1
words/internal/http/handlers/word.go:53.84,56.62 2 1
words/internal/http/handlers/word.go:56.62,60.3 3 0
words/internal/http/handlers/word.go:62.2,63.16 2 1
words/internal/http/handlers/word.go:63.16,67.3 3 1
words/internal/http/handlers/word.go:69.2,69.23 1 1
words/internal/http/handlers/word.go:73.66,75.16 2 1
words/internal/http/handlers/word.go:75.16,77.3 1 0
words/internal/http/handlers/word.go:78.2,78.14 1 1
words/internal/http/handlers/word.go:78.14,80.3 1 1
words/internal/http/handlers/word.go:82.2,82.18 1 1
words/internal/service/word/word.go:12.63,14.2 1 1
words/internal/service/word/word.go:16.51,19.2 2 1
words/internal/service/word/word.go:21.79,24.2 2 1
words/internal/storage/in_memory/in_memory.go:36.44,42.2 2 1
words/internal/storage/in_memory/in_memory.go:47.53,48.35 1 1
words/internal/storage/in_memory/in_memory.go:48.35,49.16 1 1
words/internal/storage/in_memory/in_memory.go:49.16,51.4 1 1
words/internal/storage/in_memory/in_memory.go:54.2,60.12 5 1
words/internal/storage/in_memory/in_memory.go:66.79,72.42 3 1
words/internal/storage/in_memory/in_memory.go:72.42,73.58 1 1
words/internal/storage/in_memory/in_memory.go:73.58,76.4 2 1
words/internal/storage/in_memory/in_memory.go:78.2,80.38 2 1
words/internal/storage/in_memory/in_memory.go:80.38,82.3 1 1
words/internal/storage/in_memory/in_memory.go:84.2,86.21 2 1
words/internal/storage/in_memory/in_memory.go:92.51,97.35 4 1
words/internal/storage/in_memory/in_memory.go:97.35,98.41 1 1
words/internal/storage/in_memory/in_memory.go:98.41,100.4 1 1
